.github/workflows/publish.yml

```diff
@@ -0,0 +1,25 @@
+name: Upload Python Package
```

Comment:
Please add the Apache License comment to the top of this YAML file (feel free to copy from the other workflow).

Reply:
Nit: Can you specify where the package is uploaded to? i.e. `PyPi`

Reply:
With the current configuration, the package will be uploaded on PyPi [twine](https://twine.readthedocs.io/en/stable/) takes care of looking for the PyPi's URL within the `dist` directory! You should however have a PyPi username and Password within your GitHub repository secrets.

I have just found a nice github action that looks really useful for us: https://github.com/pypa/gh-action-pypi-publish.
Under the hood, it uses twine!